tests/requirements.txt

```diff
@@ -1,3 +1,4 @@
 torch==1.7.1
 clip @ git+https://github.com/openai/CLIP.git@cfcffb90e69f37bf2ff1e988237a0fbe41f33c04
-git+https://github.com/jina-ai/jina.git@v2.0.0#egg=jina[hub]
+jina[hub]==2.0
```

Comment:
use `git+https` in the tests in order to avoid `pip install` pollution in the pypi statistics

Reply:
True, I read about that actually ..

Reply:
https://github.com/jina-ai/executor-text-clip-text-encoder/pull/4/commits/7ce62250f0c71d6f22796ae685a3e48c94e8f8c8